src/plans/create/steps/other-settings/PreserveStaticIpsField.tsx

```diff
@@ -0,0 +1,57 @@
+import type { FC } from 'react';
+import { Controller } from 'react-hook-form';
+
+import { HelpIconPopover } from '@components/common/HelpIconPopover/HelpIconPopover';
+import { Checkbox, FormGroup, FormHelperText, Stack } from '@patternfly/react-core';
+import { useForkliftTranslation } from '@utils/i18n';
+
+import { useCreatePlanFormContext } from '../../hooks';
+
+import { otherFormFieldLabels, OtherSettingsFormFieldId } from './constants';
+
+const fieldId = OtherSettingsFormFieldId.PreserveStaticIps;
+const label = otherFormFieldLabels[fieldId];
```

Comment:
Those are hardcoded values, you can include them in the component or use them implicitly

Reply:
I'm not sure what the issue is with this usage. Can you please elaborate why this should be changed?

Reply:
There isn't anything much to it, like no performance or any other reason. If you feel leaving it here is fine, as we usually add const inside the component or implicitly use it

Reply:
Updated to throw them in the components as the performance impact is ~0 anyways.